nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,81 @@
+import io
+import json
+import time
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = 'Consolidates survey data from all nodes and exports the meta graph as a JSON file.'
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--surveys-to-export',
+            type=int,
+            action='store',
+            dest='number',
+            default=1,
+            help='Number of surveys to export'
+        )
+
+        parser.add_argument(
+            '--label',
+            type=str,
+            action='store',
+            dest='label',
+            help='Optional label for the filename',
+        )
+
+    def handle(self, *args, **options):
+        """
+        Exports the latest survey data graph as a JSON file into the root directory.
+        """
+
+        number = options['number']
+        label = options['label']
+
+        streams = datastream.find_streams({'module': 'monitor.http.survey'})
+
+        # Create a meta graph
+        meta_vertices, meta_edges = [], []
+        friendly_nodes = []
+        for stream in streams:
+            datapoint_iterator = datastream.get_data(
```

Comment:
Hmm yes this is a good point. I had two uses in mind for this tool:
1. To be able to export the latest data for debugging
2. To archive graphs. This is motived by my current data collection project. So I'd like to move the rogue node 10 times and then say: find the 10 latest graphs and export them into JSON files.

How should we proceed? Is there a good way of implementing the second option?